src/javis_viewer.jl

```diff
@@ -245,27 +281,40 @@ function _javis_viewer(
         # Display image viewer
         Gtk.showall(win)
     else
-        return win, frame_dims, slide, tbox, canvas, object_list, total_frames, video
+        return win,
+        frame_dims,
+        slide,
+        tbox,
+        canvas,
+        layers,
+        object_list,
+        total_frames,
+        video
     end
-
 end
 
 """
-_jupyter_viewer(video::Video, frames::Int, actions::Vector)
+_jupyter_viewer(video::Video, frames::Int, objects::Vector{AbstractObject}; layers::Vector{Layer})
```

Comment:
You know, something to think about in the future is adding type definitions to some of our functions to improve overall computation speed... Just something to think about. Otherwise, I am glad you are adding these type definitions.